wproofreader/values.yaml

```diff
@@ -52,22 +56,37 @@ image:
   # overrides the image tag whose default is the chart appVersion
   tag: ""
 
+# Readiness Options allows customizing the default readiness probe
+readinessProbeOptions: {}
+#  initialDelaySeconds: 10
+#  periodSeconds: 10
+#  timeoutSeconds: 5
+#  successThreshold: 1
+#  failureThreshold: 3
+
+# Liveness Options allows customizing the default liveness probe
+livenessProbeOptions: {}
+#  periodSeconds: 10
+#  timeoutSeconds: 3
+#  failureThreshold: 1
+
 # number of Pods (AppServer instances) to retain; is ignored if HorizontalPodAutoscaling is enabled (see below)
 replicaCount: 1
 
 # HorizontalPodAutoscaling parameters. For more info, see documentation: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale
 # if enabled, requires "resources" (see below)
-autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 5
-  targetCPUUtilizationPercentage: 80
-  targetMemoryUtilizationPercentage: 80
-
-# max resources usage allowed for pod
+# Note: Wproofreader is able to scale only based on CPU usage metric. The targetMemoryUtilizationPercentage is not supported.
```

Comment:
```suggestion
# Note: WProofreader Server can be scaled only based on CPU usage metric. The targetMemoryUtilizationPercentage is not supported.
```

Reply:
We need to move it to the main readme as well.

Reply:
Updated README with "Application scaling" part